cli/docker/docker.go

```diff
@@ -26,6 +26,34 @@ func ConnectContainerToDevNetwork(containerID string, aliases ...string) error {
 		})
 }
 
+// ExecCommandIntoContainer executes a command, as a user, into a container
+func ExecCommandIntoContainer(containerName string, user string, cmd []string) error {
+	dockerClient := getDockerClient()
+
+	response, err := dockerClient.ContainerExecCreate(
+		context.Background(), containerName, types.ExecConfig{
+			User:         user,
+			Tty:          false,
+			AttachStdin:  false,
+			AttachStderr: false,
+			AttachStdout: false,
+			Detach:       true,
+			Cmd:          cmd,
+		})
+
+	if err != nil {
+		return err
+	}
+
+	err = dockerClient.ContainerExecStart(
+		context.Background(), response.ID, types.ExecStartCheck{
+			Detach: true,
```

Comment:
What does `Detach` mean here? Is the command executed without waiting for finalization?

Reply:
It tells Docker client to run the container with `--detach` flag (as in _docker run -d ..._), so it will run the container and will continue its execution

Reply:
Ok, so we are using this exec to start the container.

Reply:
Wait, my previous comment was misleading, sorry.

The `ContainerExecStart` method is to launch a command inside a running container, so the `Detach` attribute is to run the command in the background (Please read this: https://docs.docker.com/engine/reference/commandline/exec/#options)

Reply:
Oh ok, but then don't you want to wait or the finalization of the command to see if it worked correctly?

Reply:
We have removed it, as it's not needed anymore